src/main/java/com/auth0/json/mgmt/tenants/Tenant.java

```diff
@@ -273,10 +276,30 @@ public Integer getSessionLifetime() {
     /**
      * Setter for the login session lifetime. This is how long the session will stay valid. Value is in hours.
      *
-     * @param sessionLifetime the session life time in hours to set.
+     * @param sessionLifetime the session lifetime in hours to set.
      */
     @JsonProperty("session_lifetime")
     public void setSessionLifetime(Integer sessionLifetime) {
         this.sessionLifetime = sessionLifetime;
     }
-}
\ No newline at end of file
+
+    /**
+     * Getter for the login session idle lifetime. This is how long the session will stay valid without user activity. Value is in hours.
+     *
+     * @return the session idle lifetime in hours.
+     */
+    @JsonProperty("idle_session_lifetime")
+    public Integer getIdleSessionLifetime() {
+        return idleSessionLifetime;
+    }
+
+    /**
+     * Setter for the login session idle lifetime. This is how long the session will stay valid without user activity. Value is in hours, and decimals are allowed (for 30 minutes, use 0.5).
```

Comment:
```suggestion
     * Setter for the login session idle lifetime. This is how long the session will stay valid without user activity. Value is in hours.
```

Remove the part about decimals being allowed, since the arg is an int and the API responds with a 400 if sending a decimal.